static/js/activity.js

```diff
@@ -246,6 +246,7 @@ exports.build_user_sidebar = function () {
 };
 
 function do_update_users_for_search() {
+    popovers.hide_all();    // Hide user detail popover when the user is searching.
```

Comment:
Stylistically, we put comments on the line before something, not at the end of a line.

Reply:
@timabbott I will refactor the code. Is the logic satisfactory?

Reply:
I update the code.